docs/faq.rst

```diff
@@ -719,6 +719,18 @@ rather not implement progress / ETA display than doing futile attempts.
 See also: https://xkcd.com/612/
 
 
+Why am I getting 'Operation not permitted' errors when backing up on sshfs?
+-----------------------------------------------------------------------------
```

Comment:
iirc the underlining's length must match the text above.

Reply:
It worked when I built the docs, I can fixup anyway...